src/libraries/System.Net.Sockets/tests/FunctionalTests/TelemetryTest.cs

```diff
@@ -168,6 +168,11 @@ public void EventSource_SocketConnectsRemote_LogsConnectStartStop(string connect
         [MemberData(nameof(SocketMethods_WithBools_MemberData))]
         public void EventSource_SocketConnectFailure_LogsConnectFailed(string connectMethod, bool useDnsEndPoint)
         {
+            if (useDnsEndPoint)
+            {
+                throw new SkipTestException("https://github.com/dotnet/runtime/issues/43931");
```

Comment:
```suggestion
                // [ActiveIssue("https://github.com/dotnet/runtime/issues/43931")]
                throw new SkipTestException("https://github.com/dotnet/runtime/issues/43931");
```

So it can be found when searching for all the active issues.